docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+TestCafe command line interface is pretty simple.
+It consists of a single command that runs tests with a number of options.
+
+#### Synopsis
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+#### Browser List
+
+Specify the list of browsers where tests will be ran.
+You can specify a browser alias or path.
+
+To run tests against all installed browsers, use the *all* alias.
+
+To use remote browser connection (to run tests on a remote device),
+specify "remote" as a browser alias. If you want to connect multiple devices,
+prefix alias with the number of browsers you want to connect (e.g. "3remote").
+
+#### File Path/GLOB Pattern
+
+You can specify one or more file path or globbing pattern to run tests from.
+
+#### Options
+
+**-h**  
+**-help**  
+Display usage information for commands.
+
+**-b**  
+**--list-browsers**  
+Lists all browsers available on your system.
+
+**-r \<name\>**  
+**--reporter \<name\>**  
+Specify the reporter (*json*, *list*, *minimal*, *spec*, *xunit*) that will be
+used to generate test reports.
```

Comment:
It can be any plugin-reporter, not only built-in. E.g. if you have `testcafe-reporter-my-reporter` installed yo can use it by calling testcafe with `--reporter my-reporter` option.